### PR TITLE
AMP cleaning

### PR DESCRIPTION
**Pull request by shtukas**
## What does this change?

- Remove unused file ampYoutube.scala.html
- Rename `AmpSoundcloud` into `SoundcloudHelper`

### CODE REVIEW

File: common/app/views/support/cleaner/SoundcloudHelper.scala [hunk 1]
```diff
@@ -7,7 +7,7 @@ import org.jsoup.nodes.{Document, Element}
 
 // There are two element types that have been found to contain Soundcloud embeds.
 // These are: element-audio and element-embed
-object AmpSoundcloud {
+object SoundcloudHelper {
```
Comment (nicl) [inline common/app/views/support/cleaner/SoundcloudHelper.scala]: Hey like that this is now an object! Can we move it into the dotcomponents folder as it is only used there/relevant to that now?

**Review by nicl — COMMENTED** [review]
Hey thanks! Just a comment on moving the soundcloud helper into the dotcomponents directories somewhere.

Also, might be worth looking for any other AMP files as I can see a few just by searching for amp* files that look possibly unused now (AmpAd and AmpTest for example)? And also stuff like the AMP validator js package and working-with-amp doc perhaps?

File: common/app/views/support/cleaner/SoundcloudHelper.scala (hunk 1 above)
Comment (shtukas) [reply common/app/views/support/cleaner/SoundcloudHelper.scala]: Sure. You are right it's only used by `dotcomrendering`, didn't notice that.

**Comment by shtukas** [comment]
> Also, might be worth looking for any other AMP files as I can see a few just by searching for amp* files that look possibly unused now (AmpAd and AmpTest for example)? And also stuff like the AMP validator js package and working-with-amp doc perhaps?

Will do that in a separate PR (later today)

**Review by nicl — APPROVED** [review]
👍 💯

**Pull request closed without merging**